### DISCUSSION TITLE
Pesquisa pelo nome do campo

### DISCUSSION THREAD

**Discussion by Jefresongil** [sepinf-inc/IPED#1924] [General]
Procurei por informações sobre a sintaxe que pode ser utilizada na barra de pesquisas através do link mencionado na ajuda do programa.
No tópico "Especificaçoes de campo"menciona que "a maioria das clausulas pode ser prefixada por um nome de campo e dois pontos" 

A pergunta é: Pode ser especificado qualquer um dos campos ou somente alguns. Ao menos nas tentativas que fiz não me trouxe resultado, como por exemplo tentei especificar Communication: e nenhum resultado foi apresentado.

Abçs

**Comment by lfcnassif** [comment]
Hello. Any field can be specified. `Communication:` is not a field, but a common field prefix. The actual fields are `Communication\:From`, `Communication\:To`, and `Communication\:Date`, for example. You can also type a field prefix, like `Communication` and press the TAB key multiple times, it will autocomplete with the available field names starting with the typed prefix.

**Reply by Jefresongil** [reply]
Now it's clear. Very usefull.
